x-pack/elastic-agent/pkg/fleetapi/checkin_cmd.go

```diff
@@ -22,6 +22,7 @@ const checkingPath = "/api/fleet/agents/%s/checkin"
 // CheckinRequest consists of multiple events reported to fleet ui.
 type CheckinRequest struct {
 	Status   string              `json:"status"`
+	AckToken string              `json:"ack_token"`
```

Comment:
Should we omit if empty? otherwise we probably need to add the property to Kibana

Reply:
Sure. Will update.